Sources/AppcuesKit/Presentation/Debugger/ScreenCapturing/UIKitElementTargeting.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  UIKitElementTargeting.swift
+//  AppcuesKit
+//
+//  Created by James Ellis on 3/20/23.
+//  Copyright © 2023 Appcues. All rights reserved.
+//
+
+import UIKit
+
+// UIKit selector implementation that uses accessibilityIdentifier,
+// accessibiltyLabel, and tag data from the underlying UIView to identify elements.
+internal class UIKitElementSelector: AppcuesElementSelector {
+
+    let accessibilityIdentifier: String?
+    let accessiblityLabel: String?
```

Comment:
yes thank you. Interestingly, the base class already has `accessibilityLabel` since it inherits from NSObject, which adds that. I also discovered that we need to implement Encodable directly in the derived type to make inherited Encodable work as expected (`CodingKeys` and `encode(to encoder: Encoder)`)

in general, having to make this a derived class with NSObject in the hierarchy just for `@objc` compat is awkward, but I've yet to find a better way.

Reply:
Oh I wouldn't have expected that. Interesting